src/Pusher.php

```diff
@@ -117,7 +63,7 @@ public function __construct($auth_key, $secret, $app_id, $options = array(), $ho
             $this->settings['host'] = $host;
 
             $this->log('Legacy $host parameter provided: '.
-                                    $this->settings['scheme'].' host: '.$this->settings['host']);
+            $this->settings['scheme'].' host: '.$this->settings['host']);
```

Comment:
Fixed this in https://github.com/pusher/pusher-http-php/pull/139/commits/da5aed654494a2e78b8374a0a006ba6b5c5c3978